web/src/components/AppStoreBanner.tsx

```diff
@@ -0,0 +1,111 @@
+import { EmapsIcon } from 'icons/emapsIcon';
+import { X } from 'lucide-react';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import trackEvent from 'utils/analytics';
+
+import { isAndroid, isIphone } from '../features/weather-layers/wind-layer/util';
+import { Button } from './Button';
+
+const IS_APP_BANNER_DISMISSED = 'isAppBannerDismissed';
+
+export enum AppStoreURLs {
+  APPLE = 'https://apps.apple.com/us/app/electricity-maps/id1224594248',
+  GOOGLE = 'https://play.google.com/store/apps/details?id=com.tmrow.electricitymap&hl=en',
+}
+
+enum TrackEvents {
+  APP_BANNER_CTA_CLICKED = 'App Banner CTA Clicked',
+  APP_BANNER_DISMISSED = 'App Banner Dismissed',
+}
+
+export interface AppStoreBannerProps {
+  CloseButton?({ onClose }: DefaultCloseButtonProps): React.ReactNode;
+}
+
+export function AppStoreBanner({
+  CloseButton = DefaultCloseButton,
+}: AppStoreBannerProps) {
+  const { appStoreUrl, closeBanner } = useAppStoreBannerState();
+  const { t } = useTranslation();
+  return (
+    appStoreUrl && (
+      <div
+        role="banner"
+        aria-live="polite"
+        className="sticky flex h-14 min-h-14 w-full items-center border-b-[1px] border-solid border-neutral-300 bg-neutral-100 px-3"
+      >
+        <CloseButton onClose={closeBanner} />
+        <div className="flex flex-grow gap-2">
+          <div className="items-center justify-center self-center rounded-md border-[1px] border-neutral-200 bg-white">
+            <EmapsIcon size={40} />
+          </div>
+          <div className="content-center font-inter text-neutral-600">
+            <div className="text-sm font-semibold">Electricity Maps</div>
+            <div className="text-xs">{t('app-banner.description')}</div>
```

Comment:
```suggestion
            <p className="text-xs">{t('app-banner.description')}</p>
```
Slightly better accessibility.